frontend/views/containers/modals/JoinGroup.vue

```diff
@@ -0,0 +1,216 @@
+import sbp from '~/shared/sbp.js'
+import SvgAccess from '@svgs/access.svg'
+import SvgInvitation from '@svgs/invitation.svg'
+import SvgProposal from '@svgs/proposal.svg'
+
+export default {
+  name: 'JoinGroupModal',
+  components: {
+    ModalBaseTemplate,
+    SvgAccess,
+    SvgInvitation,
+    SvgProposal
+  },
+  data () {
+    return {
+      indicator: 0,
+      config: [
+        'get-an-invitation',
+        'wait-for-you-group-vote',
+        'use-your-unique-access-link'
+      ]
+    }
+  },
+  // TODO move this into slider component if we have more than one
+  directives: {
+    observer: {
+      inserted: (el, { value, arg }) => {
+        try {
+          const io = new window.IntersectionObserver(
+            (elements) => {
+              if (elements[0].intersectionRatio >= 0.5) { // Wait for the middle of the screen
+                value(arg)
+              }
+            },
+            { threshold: [0.5] }
+          )
+
+          io.observe(el)
+
+          el.stop = () => {
+            io.disconnect()
+          }
+        } catch (error) {
+          console.error(error.message, '\n polyfill: https://github.com/w3c/IntersectionObserver')
+        }
+      }
+    },
+    unbind (el) {
+      el.stop()
+    }
+  },
+  methods: {
+    showLoginModal () {
+      sbp('okTurtles.events/emit', REPLACE_MODAL, 'CreateGroup')
+    },
+    updateIndicator (i) {
+      this.indicator = i
+      history.pushState(null, null, `#${this.config[i]}`)
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "../../../assets/style/_variables.scss";
+
+.modal {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: $general_2;
+}
+
+.wrapper {
+  width: 100%;
+  margin: 0 auto;
+
+  @include desktop {
+    width: auto;
+  }
+}
+
+.slider {
+  display: flex;
+  overflow-x: auto;
+  -ms-overflow-style: none;
+  overflow: -moz-scrollbars-none;
+
+  /* make it smooth on iOS */
+  -webkit-overflow-scrolling: touch;
+  scroll-snap-points-x: repeat(100vw);
+  scroll-snap-type: x mandatory;
+
+  &::-webkit-scrollbar {
+    width: 0 !important;
+  }
+}
+
+.slide {
+  scroll-snap-align: center;
+  flex-shrink: 0;
+  width: 100%;
+  padding: 0.75rem;
+  text-align: center;
+
+  @include desktop {
+    text-align: left;
+    width: 16rem;
+  }
+
+  p {
+    max-width: 16rem;
+    margin: 1rem auto 0 auto;
+  }
+
+  .link {
+    margin-bottom: 1rem;
+  }
+}
+
+.slide-img {
+  display: flex;
+  align-items: center;
+  height: 6rem;
+  margin: 2.5rem auto;
+
+  svg {
+    width: 100%;
+    height: 100%;
+  }
+
+  @include desktop {
+    margin-left: 0;
+    margin: 3.5rem auto;
+  }
+}
+
+.dots {
+  display: flex;
+  justify-content: center;
+
+  @include desktop {
+    display: none;
+  }
+}
+
+.dot {
+  background-color: $general_0;
+  width: 0.5rem;
+  height: 0.5rem;
+  border-radius: 50%;
+  overflow: hidden;
+  margin: 0.25rem;
+
+  &.active {
```

Comment:
To keep consistency, when using classes that reflect state we have been using `is-` as prefix. `is-active` in this case.